Thanks for the PR!

build fails with:
```
Error: src/utils/cache/CrossOriginStorageCache.js(115,13): error TS2353: Object literal may only specify known properties, and 'origins' does not exist in type 'CrossOriginStorageRequestFileHandleOptions'.
 ELIFECYCLE  Command failed with exit code 1.
/home/runner/_work/transformers.js/transformers.js/packages/transformers:
 ERR_PNPM_RECURSIVE_RUN_FIRST_FAIL  @huggingface/transformers@4.2.0 build: `node scripts/build.mjs && pnpm typegen`
Exit status 1
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```

seems that we should update types?

--- a/packages/transformers/src/utils/cache/cross-origin-storage.d.ts
+++ b/packages/transformers/src/utils/cache/cross-origin-storage.d.ts
@@ -17,6 +17,15 @@ interface CrossOriginStorageRequestFileHandleHash {
  */
 interface CrossOriginStorageRequestFileHandleOptions {
     create?: boolean;
+    /**
+     * Restricts (or opens up) which origins may later read the stored file.
+     * `'*'` makes the resource available to all origins; an array of origin
+     * strings restricts it to that set; omitting the field keeps the
+     * default (same-site only). Visibility can be upgraded but never
+     * downgraded.
+     * @see https://github.com/WICG/cross-origin-storage#example-restricting-resources-to-specific-origins
+     */
+    origins?: '*' | string[];
 }
 
 /**

